docs/blog/2018-02-26-documentation-project/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Gatsby’s Documentation Project
+author: Shannon Soper
+date: 2018-02-22
+---
+
+Generally, people really like Gatsby docs and Gatsby in general. In fact, we’ve talked about tracking the number of curse words and endearingly pejorative terms that people use when tweeting about Gatsby. Here’s a sampling:
+
+>”HOLY SHIT @gatsbyjs && @NetflifyCMS where have you been all my life”
```

Comment:
Embed the tweets instead? Makes them more authentic + context. Also people might recognize the tweet authors.

Reply:
+ always take screenshots of the tweets just in case account gets deleted or tweet deleted.  save the screenshots in a side folder somewhere